### PR TITLE
Update Elyra image to v2.0.1

### PR DESCRIPTION
**Pull request by akchinSTC**
- Adds support for Tekton in Kubeflow Pipelines

Dependent on PR: https://github.com/opendatahub-io/s2i-lab-elyra/pull/16

### CODE REVIEW

**Comment by crobby** [comment]
This likely needs a rebase to get the tests passing.  There was a change to airflow that forced us to change some of the tests.

**Comment by vpavlin** [comment]
The PR https://github.com/opendatahub-io/s2i-lab-elyra/pull/16 need to be merged first and a release needs to be created before we merge this

**Review by harshad16 — COMMENTED** [review]
/lgtm

https://quay.io/repository/thoth-station/s2i-lab-elyra?tag=v0.0.6&tab=tags

**Comment by anishasthana** [comment]
The test is failing with:
```
ERROR:root:Message: no such element: Unable to locate element: {"method":"css selector","selector":"[id="s2i-minimal-notebook:v0.0.7"]"}
```

**Comment by nakfour** [comment]
@akchinSTC can you please rebase, we have a Z-stream release scheduled to start tomorrow, if this needs to make it in, it has to merge today.

**Review by nakfour — APPROVED** [review]

**Comment by akchinSTC** [comment]
@nakfour - thanks for the quick review!

**Comment by anishasthana** [comment]
@harshad16 are we good to unhold the PR?

File: jupyterhub/notebook-images/overlays/build/elyra-notebook-buildconfig.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
     sourceStrategy:
       from:
         kind: ImageStreamTag
```
Comment (harshad16) [inline jupyterhub/notebook-images/overlays/build/elyra-notebook-buildconfig.yaml:19]:
```suggestion
        kind: DockerImage
```

**Review by harshad16 — CHANGES_REQUESTED** [review]
the pr can be unheld.
just a small change to be made.

File: jupyterhub/notebook-images/overlays/build/elyra-notebook-buildconfig.yaml (hunk 1 above)
Comment (akchinSTC) [reply jupyterhub/notebook-images/overlays/build/elyra-notebook-buildconfig.yaml]: updated

**Review by harshad16 — COMMENTED** [review]
/lgtm

**Pull request merged**